content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -45,6 +45,37 @@ check the documentation for that version.
 
 
 <!-- body -->
+## Install and configure prerequisites
+
+These instructions apply to Linux nodes only.
```

Comment:
Do all container runtimes need this (even on Linux)? My gut feeling is they don't.

As a nice-to-have extra, explain that these steps are commonly needed but more esoteric container runtimes (I'm thinking kata containers and the like, or even a partitioning hypervisor runtime) might need a different setup.

Reply:
Oh, maybe they don't _need_ this. But the kubeadm installer will [expect](https://github.com/kubernetes/kubernetes/blob/a750d8054a6cb3167f495829ce3e77ab0ccca48e/cmd/kubeadm/app/preflight/checks.go#L1030) it for a linux node regardless of the runtime.

SHould I explain that these settings are required for preflight checks in kubeadm?

Reply:
AFAIK Docker and containerd need the forwarding and netfilter. Overlay maybe optional in some cases. Not sure about crio overall.

These things are sparsly undocumented on kubelet and CR side.

It's not kubeadm that's needs these settings but rather the kubelet will fail foif they are not set.

So we can add a note that users can refer to the CR Foo documentation if they don't want to enable something and that these are common settings..